README.md

```diff
@@ -2,6 +2,10 @@
 
 A migration tool does migrations for Konflux pipelines.
 
+## Run tests
+
+`tox`
+
```

Comment:
Could you add some info about how to create the development environment?

I suspect it would be something like this

```shell
python3 -m venv venv
source venv/bin/activate
pip install pip-tools
pip install -r requirements.txt -r requirements-test.txt
pip install -e .
```

Or maybe add a Makefile command to create a venv?

Reply:
Updated. PTAL.